scripts/gen_eval_files.lua

```diff
@@ -290,8 +282,18 @@ end
 --- Ensure code blocks have one empty line before the start fence and after the closing fence.
 ---
 --- @param x string
+--- @param special string?
+---                | 'see' Normalize for `@see`
 --- @return string
-local function norm_text(x)
+local function norm_text(x, special)
+  if special == 'see' then
+    -- @see works better without backticks and parens (for functions).
+    -- "|foo()|" => "foo"
+    -- XXX: LuaLS renders unknown symbols as strikethrough: https://github.com/LuaLS/lua-language-server/blob/8c32d8f95f28f8bc064533031898abfb7dea58b0/script/core/hover/description.lua#L144
+    x = x:gsub([=[|([^() ]+)%(%)|]=], '%1')
```

Comment:
blocked by https://github.com/LuaLS/lua-language-server/issues/2889 , unless we do some hacks in https://github.com/neovim/neovim/blob/056009f74146b349c66790fbcba3130e85a6c6da/runtime/lua/vim/lsp/util.lua#L1372

Reply:
Using tilde for strike through is GitHub flavoured markdown (as opposed to commonmark), which incidentally is also the reason tilde strike through doesn't work in matrix/element.

I'm not sure which flavour of markdown LSP allows.